File: Mobile Apps/PhysioIT/Android/src/Common/PhysioITTables.java
package Common;

import java.util.Date;



public class PhysioITTables {


    public static class FeedbackDB
    {
        public int id;
        public String Name;
        public String Email;
        public String Comments;
        public String DeviceId;
        public Date TimeStamp;
    }
    public static class DeviceDB
    {
        public int id;
        public String DeviceID;
        public String DeviceName;
        public String AppVersion;
        public String XMLVersion;
        public Date TimeStamp;
    }

    public static class BodyPartDB
    {
        public int id;
        public int PartID;
        public String Name;
        public String Image;
        public String Message;
        public Date TimeStamp;
    }

    
    public static class ReasonsDB
    {
        public int id;
        public int PartID;
        public int ReasonsId;
        public String Title;
        public String Description;
        public String Image;
        public Date TimeStamp;
    }

    public static class RemedyDB
    {

        public int id;
        public int PartID;
        public int RemedyId;
        public String Title;
        public String Description;
        public Date TimeStamp;
    }

    public static class PostureDB
    {
        public int id;
        public int PartID;
        public int RemedyId;
        public int PostureId;
        public String Wrong;
        public String Wrong_Img;
        public String Right;
        public String Right_Img;
        public String Description;
        public Date TimeStamp;
    }

    public static class ExcerciseDB
    {        
        public int id;
        public int PartID;
        public int RemedyId;
        public int ExcerciseId;
        public String Title;
        public String Image;
        public String Description;
        public String Dosage;
        public String Precautions;
        public Date TimeStamp;
    }
}

File: Mobile Apps/PhysioIT/WindowsPhone/CommonBackstage/PhysioITTables.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SQLite;
using System.Runtime.Serialization;

namespace CommonBackstage
{
    public class FeedbackDB
    {
        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public string Name
        { get; set; }

        public string Email
        { get; set; }

        public string Comments
        { get; set; }

        public string DeviceId
        { get; set; }

        public DateTime TimeStamp
        { get; set; }
    }
    public class DeviceDB
    {
        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public string DeviceID
        { get; set; }       

        public string DeviceName
        { get; set; }

        public string AppVersion
        { get; set; }

        public string XMLVersion
        { get; set; }

        public DateTime TimeStamp
        { get; set; }
    }

    public class BodyPartDB
    {
        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public int PartID
        { get; set; }

        public string Name
        { get; set; }

        public string Image
        { get; set; }

        public string Message
        { get; set; }

        public DateTime TimeStamp
        { get; set; }
    }

    public class ReasonsDB
    {

        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public int PartID
        { get; set; }

        public int ReasonsId
        { get; set; }

        public string Title
        { get; set; }

        public string Description
        { get; set; }

        public string Image
        { get; set; }


        public DateTime TimeStamp
        { get; set; }
    }

    public class RemedyDB
    {

        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public int PartID
        { get; set; }

        public int RemedyId
        { get; set; }

        public string Title
        { get; set; }

        public string Description
        { get; set; }

        public DateTime TimeStamp
        { get; set; }
    }

    public class PostureDB
    {

        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public int PartID
        { get; set; }

        public int RemedyId
        { get; set; }

        public int PostureId
        { get; set; }

        public string Wrong
        { get; set; }

        public string Wrong_Img
        { get; set; }

        public string Right
        { get; set; }

        public string Right_Img
        { get; set; }

        public string Description
        { get; set; }

        public DateTime TimeStamp
        { get; set; }

    }

    public class ExcerciseDB
    {

        [PrimaryKey, AutoIncrement]
        public int? id
        { get; set; }

        public int PartID
        { get; set; }

        public int RemedyId
        { get; set; }

        public int ExcerciseId
        { get; set; }

        public string Title
        { get; set; }

        public string Image
        { get; set; }

        public string Description
        { get; set; }

        public string Dosage
        { get; set; }

        public string Precautions
        { get; set; }

        public DateTime TimeStamp
        { get; set; }
    }

}
